pom.xml

```diff
@@ -279,6 +279,12 @@
 				<scope>test</scope>
 				<classifier>tests</classifier>
 			</dependency>
+			
+			<dependency>
+				<groupId>org.json</groupId>
```

Comment:
Do we really need this dependency?

Reply:
yes , the Growth chart API uses the JsonArray object that depends on that dependence

see `JSONArray bmis = new JSONArray();`